### ISSUE TITLE
Using highlight.js with ace / brace

### ISSUE THREAD

**Issue by akontsevich** [thlorenz/brace#144]
If some language is absent in https://github.com/thlorenz/brace/tree/master/mode is it possible to use [highlight.js](https://highlightjs.org/) somehow to highlight specified language in Brace/Ace Editor? Or need to create mode file for the language? I'm actually talking about QML.

**Comment by StoneCypher** [comment]
The way to handle it would be to add the language to ACE then to update Brace

**Comment by akontsevich** [comment]
Yes, created ticket there as well: https://github.com/ajaxorg/ace/issues/3809 There is QML module already: https://github.com/Parnswir/ace-mode-qml/ - just need to add it to ace and update brace which seems was not updated for about a 1 year.

**Comment by StoneCypher** [comment]
I'm in the middle of trying to get that done now :)

#153
